api/app/weather_models/process_grib.py

```diff
@@ -183,6 +265,61 @@ def store_bounding_values(self,
         session.add(prediction)
         session.commit()
 
+    def process_env_can_grib_file(self, session: Session, dataset, grib_info: ModelRunInfo,
+                                  prediction_run: PredictionModelRunTimestamp):
+        # for GDPS, RDPS, HRDPS models, always only ever 1 raster band in the dataset
+        raster_band = dataset.GetRasterBand(1)
+        # Iterate through stations:
+        for (points, values) in self.yield_data_for_stations(raster_band):
+            self.store_bounding_values(
+                points, values, prediction_run, grib_info, session)
+
+    def get_raster_bands(self, dataset, grib_info: ModelRunInfo):
+        """ Returns raster bands of dataset for temperature, RH, U/V wind components, and 
```

Comment:
Is it possible to write unit tests for this function since it's new and self contained?

Reply:
I'm not sure what there is to test here - `dataset.GetRasterBand()` is a gdal function. Any suggestions?

Reply:
The tests would be that we're getting the correct raster bands based on the model run interval.

1. testModelRunIntervalAt0` would have `grib_info` that results in hitting the if (zero) condition and asserting we're getting the correct bands

2. `testModelRunInterval` would have `grib_info` that results in hitting the else condition (not zero) and asserting we're getting the correct bands

Both tests would use the same `dataset` input verifying that for the same dataset, but different `grib_info`'s we get the desired results.

Reply:
If we could save an existing GFS grib file as fixture, read it in and write these tests that would be awesome.